cmd/main.go

```diff
@@ -81,16 +93,16 @@ func main() {
 	applyDynamicConfig(&canaryConfig.DynamicCanaryConfig)
 
 	glog.Infof("Starting Strimzi canary tool [%s] with config: %+v", version, canaryConfig)
-	if canaryConfig.ActivateTracing {
 
-		tp := InitTracer(canaryConfig.JaegerExporterTracing)
-		defer func() {
+	tp := InitTracerProvider(canaryConfig.JaegerExporterTracing, canaryConfig.OtlpExporterTracing)
+	defer func() {
+		if tp != nil {
```

Comment:
Why return `nil` in the no-op provider case? I would expect the no-op provider to accept the shutdown call.

Reply:
NoOptracer is from "go.opentelemetry.io/otel/trace" package and do not have the shutdown method
NewTracerProvider from  "go.opentelemetry.io/otel/sdk/trace" have a shutdown method.
In the go.opentelemetry.io/otel/sdk/trace package there is no NoOptracer

Reply:
Ah rubbish. ok thanks for confirming.